profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -44,23 +72,131 @@ fun ProfileScreen(
+                            )
+                            Box(
+                                Modifier
+                                    .maxHeight(dimensionResource(R.dimen.user_about_height))
+                                    .padding(horizontal = LocalPaddings.current.large)
+                            ) {
+                                NestedScrollableContent { contentModifier ->
+                                    about?.let {
+                                        MarkdownDocument(
+                                            markdown = it,
+                                            // TODO: Fix typography and make this an `animiteTextStyle()`.
+                                            textStyles = m3TextStyles().copy(
+                                                textStyle = m3TextStyles().textStyle.copy(
+                                                    color = textColor
+                                                )
+                                            ),
+                                            blockQuoteStyle = animiteBlockQuoteStyle(),
+                                            codeBlockStyle = animiteCodeBlockStyle(),
+                                            modifier = contentModifier
+                                        )
+                                    }
+                                }
+                            }
+                            Spacer(Modifier.size(LocalPaddings.current.medium))
+                            UserTabs()
+                        }
+                    },
+                    contentModifier = Modifier
+                        .landscapeCutoutPadding()
+                        .padding(
+                            top = LocalPaddings.current.large,
+                            bottom = dimensionResource(coreR.dimen.navigation_bar_height)
+                        )
+                )
+            }
+            else -> Login()
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun UserTabs(modifier: Modifier = Modifier) {
+    var state by remember { mutableIntStateOf(0) }
+    val titles = listOf("About", "Anime", "Manga", "Fave", "Stats")
+
+    Column(modifier) {
+        PrimaryTabRow(
+            selectedTabIndex = state,
+            containerColor = MaterialTheme.colorScheme.background,
+            divider = {}
+        ) {
+            titles.forEachIndexed { index, title ->
+                Tab(
+                    selected = state == index,
+                    onClick = { state = index },
+                    text = {
+                        Text(
+                            text = title,
+                            overflow = TextOverflow.Ellipsis,
+                            style = MaterialTheme.typography.bodyMedium,
+                            color = MaterialTheme.colorScheme.onBackground.copy(
```

Comment:
Hoist onBackground color so there's a single value to change to update the background for this and the gradient